docs/application/dotnet/guides/nui/textfield.md

```diff
@@ -0,0 +1,125 @@
+| `ScrollThreshold`                  | Float       | Horizontal scrolling will occur if the cursor is this close to the control border. |
+| `ScrollSpeed`                      | Float       | The scroll speed in pixels per second.   |
+| `SelectionHandleImageLeft`         | PropertyMap | The image to display for the left selection handle. |
+| `SelectionHandleImageRight`        | PropertyMap | The image to display for the right selection handle. |
+| `SelectionHandlePressedImageLeft`  | PropertyMap | The image to display for the left selection handle marker. |
+| `SelectionHandlePressedImageRight` | PropertyMap | The image to display for the right selection handle marker. |
+| `SelectionHandleMarkerImageLeft`   | PropertyMap | The image to display for the left selection handle marker. |
+| `SelectionHandleMarkerImageRight`  | PropertyMap | The image to display for the right selection handle marker. |
+| `SelectionHighlightColor`          | Vector4     | The color of the selection highlight.    |
+| `DecorationBoundingBox`            | Rectangle   | The decorations (handles etc) will positioned within this area on-screen. |
+| `InputMethodSettings`              | PropertyMap | The settings to relating to the System's Input Method, Key and Value. |
+| `InputColor`                       | Vector4     | The color of the new input text.         |
+| `EnableMarkup`                     | Boolean     | Whether the mark-up processing is enabled. |
+| `InputFontFamily`                  | String      | The font's family of the new input text. |
+| `InputFontStyle`                   | PropertyMap | The font's style of the new input text.  |
+| `InputPointSize`                   | Float       | The font's size of the new input text in points. |
+| `Underline`                        | PropertyMap | The default underline parameters.        |
+| `InputUnderline`                   | String      | The underline parameters of the new input text. |
+| `Shadow`                           | PropertyMap | The default shadow parameters.            |
+| `InputShadow`                      | String      | The shadow parameters of the new input text. |
+| `Emboss`                           | PropertyMap | The default emboss parameters.           |
+| `InputEmboss`                      | String      | The emboss parameters of the new input text. |
+| `Outline`                          | PropertyMap | The default outline parameters.          |
+| `InputOutline`                     | String      | The outline parameters of the new input text. |
+| `HiddenInputSettings`              | PropertyMap | Hides the input characters and instead shows a default character for password or pin entry. |
+| `PixelSize`                        | Float       | The size of font in pixels.              |
+| `EnableSelection`                  | Boolean     | Enables Text selection.                  |
```

Comment:
Suggestion : We are on text field page now (and the table is about textfield properties), so I think it would be ok not to emphasize on text editor. What do you think?